recipe/meta.yaml

```diff
@@ -1,18 +1,19 @@
 {% set name = "y-py" %}
-{% set version = "0.6.0" %}
+{% set version = "0.5.9" %}
```

Comment:
Why not version 0.6.1?

Reply:
There's a version constraint on one of the upstream packages,  jupyter_ydoc wants <0.6.0